docs/en-US/guide/installation.md

```diff
@@ -56,11 +56,11 @@ You can also use other CDN providers.
 ```html
 <head>
   <!-- Import style -->
-  <link rel="stylesheet" href="//unpkg.com/element-plus/dist/index.css" />
+  <link rel="stylesheet" href="https://unpkg.com/element-plus/dist/index.css" />
```

Comment:
```suggestion
  <link rel="stylesheet" href="//unpkg.com/element-plus/dist/index.css" />
```

Probably should not force https. Use `//` will automatically complete the https or http prefix.

Reply:
All right, I will change it back. Just wondering why not always use https 🤔

Reply:
Using https does not affect access at all, but in theory http is faster. we just provide an example.